packages/auth/amplify_auth_cognito/example/ios/Podfile

```diff
@@ -36,14 +36,14 @@ target 'Runner' do
   target 'RunnerTests' do
     inherit! :search_paths
   end
-
-  target 'unit_tests' do
-    inherit! :complete
-  end
 end
 
 post_install do |installer|
   installer.pods_project.targets.each do |target|
     flutter_additional_ios_build_settings(target)
+    # Workaround for: https://github.com/flutter/flutter/issues/126449
+    target.build_configurations.each do |config|
+      config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '13.0'
```

Comment:
This is no longer needed, right?

Reply:
Correct. We can remove for all packages in a separate PR.